pulsar-broker/src/main/java/com/yahoo/pulsar/broker/PulsarService.java

```diff
@@ -565,41 +567,56 @@ public MessagingServiceShutdownHook getShutdownService() {
     /**
      * Derive the host
      *
-     * @param isBindOnLocalhost
-     * @return
+     * @return Hostname or IP address the service binds on.
      */
     public static String host(ServiceConfiguration config) {
```

Comment:
@merlimat is it okay to change `getHost()` to `getBindAddress()`? Also, would you like me to rebase?